Specification/Definitions/units.md

```diff
@@ -2,19 +2,19 @@ Instruction and ref specification use the following units to represent quantitie
 
 | Unit                   | Examples
 |-                       |-
-| **Acceleration**       | `meter/second^2`
+| **Acceleration**       | `meter/second^2`, `millimeter/second^2`
 | **Area**               | `meter^2`
 | **Capacitance**        | `farad`, `picofarad`
-| **ElectricPotential**  | `volt`, `millivolt`
-| **Frequency**          | `hertz`, `rpm`
-| **Length**             | `meter`, `millimeter`, `micrometer`
-| **Mass**               | `gram`, `milligram`, `microgram`
-| **Matter**             | `mole`, `millimole`, `micromole`
-| **Power**              | `watt`, `milliwatt`
-| **Pressure**           | `pascal`
+| **ElectricPotential**  | `volt`, `millivolt`, `microvolt`, `nanovolt`
+| **Frequency**          | `hertz`, `kilohertz`, `rpm`
+| **Length**             | `meter`, `millimeter`, `micrometer`, `nanometer`
+| **Mass**               | `gram`, `milligram`, `microgram`, `nanogram`
+| **Matter**             | `mole`, `millimole`, `micromole`, `nanomole`
+| **Power**              | `watt`, `milliwatt`, `microwatt`
+| **Pressure**           | `pascal`, `bar`, `torr`
 | **Temperature**        | `celsius`, `kelvin`
```

Comment:
Don't we also take in Fahrenheit/Rankine?

Reply:
hmm apparently not... at least I don't see that in our support unit spec

Reply:
Oh. Guess not all the available units are in the spec, then? Never mind. :p Thanks!